jsonclient/client.go

```diff
@@ -41,6 +44,69 @@ type JSONClient struct {
 	httpClient *http.Client          // used to interact with the server via HTTP
 	Verifier   *ct.SignatureVerifier // nil for no verification (e.g. no public key available)
 	logger     Logger                // interface to use for logging warnings and errors
+
+	multiplier int64
+	until      *time.Time
```

Comment:
Might be clearer renamed to `notBefore` (but maybe I've looked at too many certificates...)